config/grpc_client_gen.config

```diff
@@ -0,0 +1,33 @@
+{plugins, [
+    {grpcbox_plugin,
+        {git, "https://github.com/novalabsxyz/grpcbox_plugin.git",
+            {branch, "andymck/ts-master/combined-opts-and-template-changes"}}}
+]}.
+
+{grpc, [
+    {proto_files, [
+        "_build/default/lib/helium_proto/src/service/router.proto"
+    ]},
+    {out_dir, "src/grpc/autogen/client"},
+    {beam_out_dir, "src/grpc/autogen/client"},
+    {keep_beams, false},
+    {create_services, false},
+    {type, client},
+    {override_gpb_defaults, true},
+    {gpb_opts, [
+        use_packages,
+        {maps, true},
+        {defs_as_proplists, true},
```

Comment:
I'm not sure where we got this from, but if we're going to use straight `grpc_client` this setting needs to be false. When creating a stream `grpc_client_stream` assumes defs are maps, and will fail silently otherwise.

https://github.com/Bluehouse-Technology/grpc_client/blob/master/src/grpc_client_stream.erl#L229

Reply:
Fixed.